Resources/Prototypes/Entities/Objects/Devices/Circuitboards/Machine/production.yml

```
@@ -862,6 +867,7 @@
     stackRequirements:
       Capacitor: 4
       Steel: 5
+      BScrystalArt: 2
```

```suggestion
      BScrystalArt: 2 #ADT Tweak
```
No ya bi ubral vprincipe